### PR TITLE
Zowe/zowe cli/139 declarative to scripted

### PR DESCRIPTION
**Pull request by AHumanFromCA**
Introduce the initial NodeJS pipeline runner. (closes zowe/zowe-cli#139)

### CODE REVIEW

**Comment by AHumanFromCA** [comment]
@zFernand0 can you modify the JenkinsPipeline file? It says the entire file conflicts with master so not sure what you have changed.

**Comment by AHumanFromCA** [comment]
I noticed there was a problem within the generated groovy doc. I have fixed and uploaded. If we are good with the changes, I can merge and also update perf-timing.

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,68 @@
+@Library('shared-pipelines@zowe/zowe-cli/139_declarative-to-scripted')
```
Comment (ChrisBoehmCA) [inline Jenkinsfile]: should we change this to @master?

Comment (AHumanFromCA) [reply Jenkinsfile]: Actually we should get rid of the `@` entirely. The default setup is to pull master

Comment (AHumanFromCA) [reply Jenkinsfile]: I got rid of the tag. Due to jenkins constraints, this pr will have a failing commit until it is merged into master.

Comment (AHumanFromCA) [reply Jenkinsfile]: I have changed this to be a dynamic import, hopefully it will work properly for both PRs and regular build commits.

**Pull request merged**